tutorials/core-concepts/pieces/pieces-example.md

```diff
@@ -0,0 +1,242 @@
+```
+
+![](/.gitbook/assets/pieces-widget-page.png) 
+
+Congratulations! You created a new content type and a widget to display it. However, there are some limitations to using a widget to display this sort of content. One limitation is that everything you do and configure is contained within the instance of the widget placed on a specific page. Another is that the pieces themselves don’t have a “home” that you can universally link to them from. Using pieces pages will solve this.
+
+## Create the Pieces Pages
+
+Since you created the new piece type with the `--pages` options, you already have the pieces pages module created. Now, you must register the pieces pages module in `app.js`, and create the templates for the index and show pages. 
+
+![](/.gitbook/assets/pieces-pages-directory.png) 
+
+1. Add the following code to `app.js` in the main modules section to register the pieces pages module.
+  
+  ```markup
+    modules { 
+      ... 
+      'foods-pages': { 
+          extend: 'apostrophe-pieces-pages'
+        }, 
+      ...
+    }
+  ```
+
+2. Then register the pieces pages as a new page type by adding this is the `pages` subsection of `modules`:
+
+  ```markup
+    {
+      name: 'foods-pages',
+      label: 'Food’'
+    }
+  ```
+
+3. Open `index.html` and add the following code:
+
+  ```markup
+    {% extends "layout.html" %}
+     {% block title %}{{ data.page.title }}
+    {% endblock %}
+
+    {% block main %}
+      <div class="main-content">
+        {% for piece in data.pieces %}
+          <h4><a href="{{ piece._url }}">{{ piece.title }}</a></h4> ${{ piece.price }}
+        {% endfor %}
+      </div>
+    {% endblock %} \
+  ```
+
+4. Open `show.html` and add the following code:
+
+
+  ```markup
+    {% extends "layout.html" %}
+    {% block title %}{{ data.piece.title }}{% endblock %}
+    
+    {% block main %}
+      <div class="main-content">
+        <h4> {{ data.piece.title }}
+        ${{ data.piece.price }}
+        
+        {{ apos.singleton( data.piece, 'image', 'apostrophe-images',
+          { edit: false }
+        )}}
+      </div>
+    {% endblock %}
+  ```
+
+### Displaying Pieces with Pieces Pages
+
+After you create and register the pieces pages module, you can create a new page for your piece type.
+
+1. While logged into Apostrophe, open the _Page Menu_ and select _New Page_.
+
+2. Under _Page Type_ select “Food”.
+
+3. Name the page “Menu” and click _Save_.
+
+![](/.gitbook/assets/pieces-create-pieces-page.png) 
+
+This creates the piece index page. The `index.html` template generates links for each piece of the piece type.
+
+![](/.gitbook/assets/pieces-menu-page.png) 
+
+The pages for each piece are generated from the `show.html`, so you now have individual pages for each piece.
```

Comment:
I get why you keep calling these "pages," it's a bit exhausting to keep distinguishing between pages in the tree and these "virtual" pages without even having a clear term for it. So how about we just call them that consistently — virtual pages?